PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -4050,61 +4078,156 @@ Bool_t AliConvEventCuts::MimicTrigger(AliVEvent *event, Bool_t isMC ){
       }
     }
 
+
+
   // Get individual threshold for every Supermodule (if no Supermodulewise was defined threshold is the same for all SMs)
   Float_t fTriggThresh[20] = {0};
   fRandom.SetSeed(0);
-  for(int iSM = 0; iSM < fHistoTriggThresh->GetNbinsX(); iSM++){
-    fTriggThresh[iSM] = fRandom.Gaus(fHistoTriggThresh->GetBinContent(iSM + 1)*0.01, fHistoTriggThresh->GetBinError(iSM + 1)*0.01);
```

Comment:
I think you took these two lines out by accident. Or at least in the diff here I don't see where the threshold is otherwise initialized.

Reply:
solved... Nico pointed me to the new line